build.gradle

```diff
@@ -20,11 +21,24 @@ publishing {
         ivy(IvyPublication) {
             from components.java
         }
+
+    	mavenCustom(MavenPublication) {
+          pom.withXml {
+            asNode().appendNode('description',
+                                'A demonstration of maven POM customization')
+          }
+        }
+        mavenJava(MavenPublication) {
+            groupId 'org.stellar.sdk'
+            artifactId 'stellar-sdk'
+            from components.java
+        }
     }
     repositories {
         ivy {
             url "$projectDir/repos"
         }
+	mavenLocal()
```

Comment:
Can you remove changes to this file? It's not in a scope of this PR.

Reply:
will do